packages/components/package-lock.json

```diff
@@ -5035,11 +5035,6 @@
         }
       }
     },
-    "@chanzuckerberg/czedi-kit-tokens": {
```

Comment:
didn't know if/how to update this but I figure it'll get installed correctly later

Reply:
Hmmm, weird. I'd expect Lerna to do whatever it needs to do correctly.

Reply:
oh god I tried `lerna version` and it pushed a commit for me 😭 and also didn't update this correctly lol...sad

Reply:
Yeah, that's a downside to Lerna - when what you wanted to happen didn't happen, it's still going to push stuff (commits, tags, etc).

Annoyingly, there also isn't a `--dry-run` flag, which would be useful.